tailwind.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  purge: [
+    './src/pages/**/*.{js,ts,jsx,tsx}',
+    './src/components/**/*.{js,ts,jsx,tsx}',
+  ],
+  darkMode: false, // or 'media' or 'class'
```

Comment:
is there a reason why we are not planning to go for making `darkmode` option available to the user if it is simple to integrate? or are we keeping it simple for mvp?